.github/workflows/ci.yml

```
@@ -32,7 +34,7 @@ jobs:
       env:
         CACHE_FROM_IMAGE: us.gcr.io/${{secrets.GCLOUD_PROJECT_ID}}/kaori/kaori:master-latest
     - name: Run the tests
-      run: ./tests/run.sh
+      run: ./scripts/run.sh
```

```suggestion
      run: ./tests/run.sh
```
oops this was a bad merge